src/libraries/System.Data.Odbc/src/System/Data/Odbc/OdbcDataReader.cs

```diff
@@ -2344,7 +2344,6 @@ internal int RetrieveKeyInfo(bool needkeyinfo, QualifiedTableName qualifiedTable
             bool mustRelease = false;
             Debug.Assert(buffer.Length >= 264, "Native buffer to small (_buffer.Length < 264)");
 
-            RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
This project targets netstandard + netframework.
(This comment applies file-wide.)

Reply:
The file is in an ItemGroup that only targets NetCoreApp.